src/cmd/flux-jobs.py

```diff
@@ -550,6 +559,13 @@ def main():
         else:
             print(json.dumps({"jobs": result}))
 
+    # "no_header" also applies to trailers :-)
+    if not args.no_header and truncated:
+        print(f"warning: output truncated at {args.count} jobs.")
+        print(
+            "Use --count to increase returned results or use filters to alter results."
+        )
```

Comment:
A couple thoughts here:
The output of `flux jobs` could be captured, so should the warning be to stderr?

I wonder if when using `-c N` the user expects truncation, so should the warning be suppressed if `args.count` was specified by the user? The warning might be especially annoying with `-c 1` for example.

Also, then if you use this strategy of asking for N+1, it will only be used for the default=1000 case.

Reply:
> The output of flux jobs could be captured, so should the warning be to stderr?

Oh good point!

> I wonder if when using -c N the user expects truncation, so should the warning be suppressed if args.count was specified by the user? The warning might be especially annoying with -c 1 for example.

> Also, then if you use this strategy of asking for N+1, it will only be used for the default=1000 case.

Hmmmm.  That's an interesting idea.   My initial thought is to still do it when the user specifies --count.  I imagine the default scenario is they run `flux-jobs` and see the "output truncated" message.  They then increase the count to 2000 ... they need a warning if that increased count is also truncated?

Reply:
Ah good point I hadn't thought of that. I'm going to be really annoyed when I get the error message with `-c 1` though :thinking: Perhaps this will just train users to use `-n, --no-header`?

Reply:
yeah, that's why I tied the message to `--no-header`, b/c I figure anyone scripting with it is already using it?  Although I suppose some might be using `tail -n 1` with `-c 1`?

Reply:
Yeah, the warning should still go to stderr, but you're right it needs to be there on any value for `--count` that isn't zero.